package.json

```
@@ -0,0 +1,47 @@
+{
+  "name": "dotflow-ui",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "start": "next dev",
+    "build": "next build",
+    "lint": "next lint"
```

```suggestion
    "lint": "next lint",
    "format":  "npx prettier --write '**/*.{js,jsx,ts,tsx}'"
```

Also, a CI check to run prettier would be nice.